README.md

```diff
@@ -4,6 +4,26 @@ This project is part of ASP.NET Core. You can find samples, documentation and ge
 
 Use https://github.com/aspnet/AspNetCore/issues for issues with this project.
 
+## Build this library
+
+There are multiple ways to build this library
+
+* Use [vcpkg](https://github.com/microsoft/vcpkg) and install the library with `vcpkg install microsoft-signalr`
+* Build from [command line](#command-line-build)
+* Build in Visual Studio (must have the "Desktop Development with C++" workload) by selecting the "Open a local folder" option and selecting the repository root folder
```

Comment:
The the "Desktop Development with C++" workload (or rather some part of it) is required for the command line build as well.

I also recommend linking to https://docs.microsoft.com/en-us/cpp/build/cmake-projects-in-visual-studio?view=msvc-160. Before finding that, it wasn't obvious to me that I needed to set CMakeLists.txt as the startup item to build.

Python is another required dependency for the command line (and presumably VS) build we should call out.

Reply:
Python? What for?

Reply:
When I remove python from my path and run the `cmake .. -DCMAKE_TOOLCHAIN_FILE=...` command, the output contains the following:

```
-- Could NOT find PythonInterp (missing: PYTHON_EXECUTABLE)
```

It appears it was just informational though, because I can continue building successfully despite this. I wonder why cmake is even looking for this.

I thought missing python was actually causing the build to fail earlier, but I don't still have the console output from the first time I saw the failure and I cannot reproduce it. It's possible I just saw that log and misinterpreted it as the reason for another build failure caused by something else which I later fixed.